docs/guides/development/integrations/databases/fauna.mdx

```diff
@@ -91,23 +91,9 @@ This guide will walk you through the steps to integrate Fauna with Clerk in your
 
   Add the Fauna library to your project.
 
-  <CodeBlockTabs options={["npm", "yarn", "pnpm", "bun"]}>
-    ```bash {{ filename: 'terminal' }}
-    npm i fauna
-    ```
-
-    ```bash {{ filename: 'terminal' }}
-    yarn add fauna
-    ```
-
-    ```bash {{ filename: 'terminal' }}
-    pnpm add fauna
-    ```
-
-    ```bash {{ filename: 'terminal' }}
-    bun add fauna
-    ```
-  </CodeBlockTabs>
+  ```npm
+  npm install fauna
```

Comment:
I [asked in Slack](https://clerkinc.slack.com/archives/C01QFRQNHSN/p1762979353082119), but [Fauna no longer exists as a service as of May 30, 2025](https://web.archive.org/web/20250530173054/https://fauna.com/blog/the-future-of-fauna).

I think we merge this PR with these changes, and follow up with a separate PR for a full removal of this Guide.